templates/codeowners/DEFAULT_CODEOWNER

```diff
@@ -0,0 +1,2 @@
+# Default owners are pcds-platforms-development
+* @pcdshub/pcds-platform-development
```

Comment:
Questions/follow-up ideas:
What is this file used for?
Should we rename this team?